src/modules/Controller/usePointerSwipe.ts

```diff
@@ -61,35 +73,52 @@ export function usePointerSwipe<T extends Element = Element>(
 
   const onPointerDown = useEventCallback((event: React.PointerEvent) => {
     addPointer(event);
+    // 记录起始位置
+    startPosition.current = { x: event.clientX, y: event.clientY };
+    isClick.current = true;
+    startClickTime.current = Date.now();
   });
 
   const exceedsPullThreshold = (value: number, threshold: number) =>
     (pullDownEnabled && value > threshold) || (pullUpEnabled && value < -threshold);
 
   const onPointerUp = useEventCallback((event: React.PointerEvent) => {
-    if (pointers.current.find((x) => x.pointerId === event.pointerId) && activePointer.current === event.pointerId) {
-      const duration = Date.now() - startTime.current;
-      const currentOffset = offset.current;
-
-      if (gesture.current === Gesture.SWIPE) {
-        if (
-          Math.abs(currentOffset) > 0.3 * containerWidth ||
-          (Math.abs(currentOffset) > 5 && duration < swipeAnimationDuration)
-        ) {
-          onSwipeFinish(currentOffset, duration);
-        } else {
-          onSwipeCancel(currentOffset);
-        }
-      } else if (gesture.current === Gesture.PULL) {
-        if (exceedsPullThreshold(currentOffset, 2 * SWIPE_THRESHOLD)) {
-          onPullFinish(currentOffset, duration);
-        } else {
-          onPullCancel(currentOffset);
+    if (pointers.current.find((x) => x.pointerId === event.pointerId)) {
+      const clickDuration = Date.now() - startClickTime.current;
+      // Handle click events
+      if (isClick.current && clickDuration < 300) {
+        const target = event.target as HTMLElement;
+        if (closeOnBackdropClick && target && target.classList.contains(cssClass(CLASS_SLIDE_WRAPPER))) {
```

Comment:
please add `|| target.classList.contains(cssClass("slide"))` - that's the use when neither Zoom nor Video plugins are enabled.